docs/api/app.md

```diff
@@ -228,6 +228,10 @@ app.on('login', function(event, webContents, request, authInfo, callback) {
 
 Emitted when the gpu process crashes.
 
+### Event: 'dark-mode-changed' _OS X_
```

Comment:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms632650(v=vs.85).aspx for Windows, having some trouble finding this for GTK+